eodhp-web-presence/templates/web-presence-deploy.yaml

```diff
@@ -42,3 +42,7 @@ spec:
               value: {{ .Values.catalogue_data.url | default "https://my-stac-browser-data.com/catalog.json" }}
             - name: NOTEBOOKS_URL
               value: {{ .Values.notebooks.url | default "https://notebooks.co.uk/apphub" }}
+            - name: ALLOWED_HOSTS
+              value: {{ .Values.hosting.allowed_hosts | default "localhost,127.0.0.1,my.website.co.uk" }}
+            - name: BASE_URL
+              value: {{ .Values.hosting.base_url | default "www.example.com" }}
```

Comment:
Similarly, just make this 127.0.0.1 as this is usually the default in dev. Also, does this not require http:// (https:// in prod) in front? Generally, URLs require the protocol and host names do not. From the [Wagtail docs](https://docs.wagtail.org/en/latest/reference/settings.html#wagtailadmin-base-url) it seems that it expects the protocol here.

Reply:
Updated